.github/workflows/main.yaml

```diff
@@ -33,6 +33,7 @@ jobs:
       uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
       with:
         version: latest
+        args: -v -c .golangci.yaml
```

Comment:
By default I believe the `golangci-lint` action picks up `.golangci.yml` instead of `.golanci.yaml`.